README.adoc

```diff
@@ -1,3 +1,21 @@
+= C++ example project scanned on SonarQube using Bitbucket Cloud pipelines
+
+To analyze a C/C++/Objective-C project on Bitbucket pipelines with SonarQube, you need to:
+
+. Enable pipelines in your Bitbucket Cloud repository
+. Add a repository variable `SONAR_TOKEN` with a token generated in SonarQube (see https://docs.sonarqube.org/latest/user-guide/user-token/[Generating and Using Tokens])
+. Add a repository variable `SONAR_HOST_URL` that points to your SonarQube server
+. In your `bitbucket-pipelines.yml` file:
+.. Download the https://sonarcloud.io/documentation/analysis/languages/cfamily/#analysis-steps-using-build-wrapper[`build wrapper`] and wrap your compilation with it
```

Comment:
Do you really want a SonarCloud link in a SonarQube-oriented repository?

Reply:
No, good catch